tutorials/autotvm/tune_nnvm_arm.py

```diff
@@ -257,9 +262,17 @@ def tune_tasks(tasks,
     for i, tsk in enumerate(reversed(tasks)):
         prefix = "[Task %2d/%2d] " % (i+1, len(tasks))
 
+        # if we want to use spatial pack for depthwise convolution
+        if try_spatial_pack_depthwise and tsk.name == 'topi_nn_depthwise_conv2d_nchw':
+            tsk = autotvm.task.create(tsk.name, tsk.args,
+                                      tsk.target, tsk.target_host, 'contrib_spatial_pack')
+
         # create tuner
         if tuner == 'xgb' or tuner == 'xgb-rank':
-            tuner_obj = XGBTuner(tsk, loss_type='rank')
+            if try_spatial_pack_depthwise and tsk.name == 'topi_nn_depthwise_conv2d_nchw':
```

Comment:
force the tuner setting under a single 
`if try_spatial_pack_depthwise` block

Reply:
if we do like try_winograd for loop, I can not think of how to merge this in try_spatial_pack_depthwise for loop. Because we will init tunner in this loop.

Reply:
We can do like

```python
if try_spatial_pack_depthwise:
    tuner = 'xgb_knob'
...

elif tuner == 'xgb_knob':
    tuner_obj = XGBTuner(tsk, loss_type='rank', feature_type='knob')
```

Reply:
Thanks. I understand your meaning now.

Reply:
Have applied your suggestions. @merrymercy could you help review again?